src/Workspaces/Core/Portable/Workspace/Host/DocumentService/IDocumentOperationService.cs

```diff
@@ -13,10 +13,5 @@ internal interface IDocumentOperationService : IDocumentService
         /// document version of <see cref="Workspace.CanApplyChange(ApplyChangesKind)"/>
         /// </summary>
         bool CanApplyChange { get; }
-
-        /// <summary>
-        /// indicates whether this document supports diagnostics or not
```

Comment:
I think @heejaechang should review this. I thought this can be false for our internal partners that have IVT access (TS, F#, razor, etc.). Best for @heejaechang to confirm if the API is really unused.

Reply:
➡️ Duplicate of https://github.com/dotnet/roslyn/pull/34009#event-2193856326 😄 

Currently some internal partners may set this to false, but there is no need to do this. Reporting a diagnostic is valid regardless of the location.